src/components/Slider/SliderFixed.tsx

```diff
@@ -183,7 +185,14 @@ const SliderFixed: React.FC<SliderProps> = ({
             onPressPoint={(point: number) => tapToSeek && onPressPoint(point)}
           />
         )}
-        <Tracked style={[trackedStyle, animatedTrackStyle]} />
+        <Track
+          style={StyleSheet.flatten([
+            styles.track,
```

Comment:
The styles.track is StyleSheet.absoluteFillObject so we do not need create this style